cfy_manager/components/restservice/restservice.py

```diff
@@ -355,20 +410,74 @@ def _upload_cloudify_license():
 
     def install(self):
         logger.notice('Installing Rest Service...')
-        self._install()
+        yum_install(config[RESTSERVICE][SOURCES]['restservice_source_url'])
+        yum_install(config[RESTSERVICE][SOURCES]['agents_source_url'])
+
+        self._chown_resources_dir()
+
+        set_logrotate(RESTSERVICE)
+
+        if DATABASE_SERVICE not in config[SERVICES_TO_INSTALL]:
+            yum_install(config[RESTSERVICE][SOURCES]['haproxy_rpm_url'])
         logger.notice('Rest Service successfully installed')
 
     def configure(self):
         logger.notice('Configuring Rest Service...')
-        self._configure()
+        if (
+            DATABASE_SERVICE not in config[SERVICES_TO_INSTALL]
+            and config[POSTGRESQL_CLIENT]['host'] in (
+                '127.0.0.1', 'localhost',
```

Comment:
maybe we can allow empty a well?

Reply:
I don't think that's allowed to be empty? It needs to be set to localhost or 127.0.0.1 at least, because otherwise the places where we use it in the, e.g. restservice config won't be able to hit the DB since they'll not have a host specified.

For psql, on the other hand, empty host == use unix socket, so that's fine (for local).